torch/quantization/fx/prepare.py

```diff
@@ -458,7 +460,10 @@ def maybe_insert_output_observer_for_node(
                     get_default_output_activation_post_process_map().get(
                         matched_pattern,
                         act_post_process_ctr)
+
             observer = act_post_process_ctr()
+            if isinstance(observer, _InputEqualizationObserver):
+                observer = observer.output_obs
```

Comment:
I feel output_obs needs to be a constructor as well (see comment in previous PR)